sass/_Omnibox.scss

```diff
@@ -5,18 +5,35 @@
   z-index: 0;
 }
 .CoveoOmnibox.magic-box {
-  .coveo-omnibox-hightlight {
+  .coveo-omnibox-suggestion-input {
     font-weight: bold;
     color: $black;
   }
   .magic-box-suggestions {
     z-index: 3;
   }
-  .coveo-omnibox-hightlight2 {
-    font-weight: bold;
+  .coveo-omnibox-suggestion-input-corrected {
     font-style: italic;
     color: $black;
   }
+  .magic-box-suggestion {
+    font-weight: bold;
+    color: $black;
+
+    .coveo-omnibox-suggestion-input {
+      font-weight: initial;
```

Comment:
The initial key-word is [not supported](https://caniuse.com/#feat=css-initial-value) in IE11. Better to use `normal` in this case.

Same comment applies to line 28.